package.json

```diff
@@ -158,7 +158,7 @@
     "react-dom": "^16.3.1"
   },
   "resolutions": {
-    "typescript-eslint-parser": "17.0.1"
+    "typescript-eslint-parser": "20.0.0"
```

Comment:
Should we (can we) get rid of this resolution?

Reply:
We could as this is what sewing-kit uses under the hood, however the next time sewing-kit changes it version of this we'll have to add it back to keep us in sync with the version of typescript that we use. I think it's worth keeping as a reminder.